docs/en/observability/uptime-set-up-app.asciidoc

```diff
@@ -22,6 +22,35 @@ the following privileges:
 From here, you can either <<global-managed-testing-infrastructure,use Elastic's global managed testing infrastructure>>
 or <<private-locations,create a private location>> to run monitors from your own premises.
 
+++++
+<div class="imageblock">
+<img
+  src="images/uptime-set-up-app.png"
+  class="w-100"
+  alt="Diagram showing which pieces of software are used to configure monitors, create monitors, and view results when using the Uptime App. Described in detail in Diagram text description."
+  aria-details="diagram-text-app">
+</div>
+++++
+
+[[diagram-text-app]]
+[%collapsible]
+.Diagram text description
+====
+. Where to start when using the {uptime-app} depends on whether you are creating a <<monitoring-uptime,lightweight monitor>> or <<monitoring-synthetics,browser monitor>>. It also depends on whether you are using a project to manage journey code.
+    * _Option 1_: For lightweight monitors and browser monitors that are _not_ managed in a project, you can configure a new monitor in **{kib}**, specifically the **{observability} {uptime-app}**.
+        ** For browser monitors configured in the **{observability} {uptime-app}**, you must include **Inline journey code**, which can be written from scratch or generated using the <<synthetics-recorder,Synthetics recorder>>.
+    * _Option 2_: For other browser monitors, start with a **Project using @elastic/synthetics** and configure monitors using `monitor.use`.
```

Comment:
```suggestion
    * _Option 2_: For other browser monitors, start an @elastic/synthetics project with `elastic-synthetics init`. Then, configure monitors using `monitor.use`.
```

I think this sounds less weird as users will probably not know what we mean by a "project using @elastic/synthetics". Instead we provide them with actionable advice.

Reply:
I want to use the exact text as it appears in the diagram, which is "Project using @elastic/synthetics". 

```suggestion
    * _Option 2_: For other browser monitors, create a **Project using @elastic/synthetics** using `elastic-synthetics init`. Then, add journey code and configure monitors using `monitor.use`.
```